cart-service/src/main/java/com/petclinic/cartsservice/dataaccesslayer/Cart.java

```diff
@@ -25,4 +25,9 @@ public class Cart {
     private List<CartProduct> products;
     private String cartId;
     private String customerId;
+    //added those
+    private double subtotal;
+    private double tvq;
+    private double tvc;
+    private double total;
```

Comment:
Could you explain me why you added those fields in the entity. I think it would be a good idea to keep those calculations in the toCartResponseModel function. 

![image](https://github.com/user-attachments/assets/97467147-4148-4e65-a4cf-5c3053ae57c3)